src/main/java/org/spongepowered/api/world/World.java

```diff
@@ -262,6 +262,14 @@ default LocatableBlock getLocatableBlock(int x, int y, int z) {
     WorldBorder getWorldBorder();
 
     /**
+     * Sets the world border for this world.
+     * 
+     * @param border The world border to be used.
+     * @param cause The cause of the world border's change.
+     */
+    void setWorldBorder(WorldBorder border, Cause cause);
```

Comment:
Internally, the border is modified and not replaced as the field that contains the world border is final. It is basically the same as modifying each aspect of the world's border with the supplied border through the `getWorldBorder().setX()` methods except through a single method call not several.